app.rb

```diff
@@ -115,4 +117,67 @@ def list_rentals
       puts 'Invalid person ID'
     end
   end
+
+  def save_data
+    save_people
+    save_books
+    save_rentals
+    puts 'Thanks For Using me'
+  end
+
+  def load_data
+    load_people
+    load_books
+    load_rentals
+  end
+
+  private
+
+  def save_people
+    File.write('people.json', JSON.pretty_generate(@people))
+  end
+
+  def save_books
+    File.write('books.json', JSON.pretty_generate(@books))
+  end
```

Comment:
- [ ] 💡 Tip: Instead of storing your data like this:

![image](https://user-images.githubusercontent.com/77069011/225160378-0219d423-3335-492e-8782-f1ef80a841ac.png)

Consider writing all the attributes of the instance, it will make things easier when you try to load your data from the JSON file and recreate your instances, at the moment, it is just a string and it's not possible to create a Book from it! 🤓 🔧